recipe/meta.yaml

```diff
@@ -10,21 +12,17 @@ build:
   number: 0
   detect_binary_files_with_prefix: true
   features:
-    - vc9     # [win and py27]
-    - vc10    # [win and py34]
-    - vc14    # [win and py35]
+    - vc{{ VC_VERSION }}   # [win]
 
 requirements:
   build:
-    - python         # [win]
     - curl           # [win]
     - 7za            # [win]
+    - vc             # [win]
```

Comment:
I recommend including the {{ VC_VERSION }} as a version pin in this line and below.  It seems to be working fine without it, but I just don't trust these things...

Happy to merge if you fix this, or if you tell me that you don't want to.